### DISCUSSION TITLE
Filters problem

### DISCUSSION THREAD

**Discussion by crawe1** [YetiForceCompany/YetiForceCRM#14923] [How to] [answered]
Hi,
I have a problem with filter whats i do wrong ?
I install new module and add in this module filter :
![image](https://user-images.githubusercontent.com/61950059/124670329-1e5fb380-deb4-11eb-8d4f-a726f913aec1.png)
and i want to see this columns from filter in relation table
![image](https://user-images.githubusercontent.com/61950059/124670607-81e9e100-deb4-11eb-84d9-fdafe9c7483c.png)
but i see only 2 columns
![image](https://user-images.githubusercontent.com/61950059/124670684-a219a000-deb4-11eb-9e33-9b6c4cb7328e.png)
filters on standard module works fine but filter in new install module not work only (name and owner) where is problem ?

**Comment by rskrzypczak** [comment, marked as answer]
Go to the main module file and empty the $search_fields_name variable.
Here is an example: https://github.com/YetiForceCompany/YetiForceCRM/commit/ae852816707a8a2140ee9c64cc4dd688d1c37f27

**Reply by crawe1** [reply]
thank u but do you know where is function whats generate this file ?

**Reply by rskrzypczak** [reply]
I assume you are using an old version of the system
https://github.com/YetiForceCompany/YetiForceCRM/commit/ae852816707a8a2140ee9c64cc4dd688d1c37f27#diff-39c962982f80d3823f5f67fd6fd3b2edfeda68a84356b4710739bd3c9ca22975L59

**Reply by crawe1** [reply]
yes but your answer helped me, now it working. yest i use 5.3 maybe 5.6  :)
